framework/test_knobs.cpp

```diff
@@ -85,9 +85,11 @@ void clear_test_knobs(){
 
 // external interface methods
 
-const char *get_testspecific_knob_value_string(const struct test *test, const char *key,
-                                               const char *value_if_not_present)
+const char *get_testspecific_knob_value_string(const struct test *test, const char *key, const char *value_if_not_present)
 {
+#if SANDSTONE_I18N_LOGGING
```

Comment:
This should be `SANDSTONE_RESTRICTED_CMDLINE` or (better) an `if constexpr (SandstoneConfig::RestrictedCommandLine)`

Reply:
> This should be `SANDSTONE_RESTRICTED_CMDLINE` or (better) an `if constexpr (SandstoneConfig::RestrictedCommandLine)`

Ah, good call.